src/authentication/authentication-hapi-plugin.ts

```diff
@@ -267,3 +303,9 @@ export function generatePassword(length = 16) {
 export function sanitizeUsername(username: string) {
   return username.replace('|', '-');
 }
+
+export function assertTeam(teamNameOrId: string |  number, team: Group) {
+  return team.name.toLowerCase() === teamNameOrId.toString()
+    || team.path === teamNameOrId.toString()
```

Comment:
With the command line tool we'll probably like to use the team name and to be able to include it in the path like this `/team-token/{teamIdOrName?}` it has to pathified.